compose/frontend.yml

```diff
@@ -83,13 +83,15 @@ services:
       PROXY_DEFINITION_IMPORT: http://ccd-definition-store-api:4451
       PROXY_DEFINITION_DATA: http://ccd-definition-store-api:4451/api/data
       PROXY_DEFINITION_DISPLAY: http://ccd-definition-store-api:4451/api/display
+      PROXY_CASE_DOCUMENT: http://ccd-case-document-am-api:4455
```

Comment:
Is this still part of the plan given [RDM-11417](https://tools.hmcts.net/jira/browse/RDM-11417?focusedCommentId=970009&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-970009) has been **WITHDRAWN**?

Reply:
Removed

Reply:
Fixed in 889a833